cloudinit/distros/gentoo.py

```diff
@@ -261,5 +267,13 @@ def convert_resolv_conf(settings):
             result += "nameserver %s\n" % ns
     return result
 
+def convert_resolv_conf_search(settings):
+    """Returns a settings string formatted for resolv.conf."""
+    result = ""
+    if isinstance(settings, list):
+        for domain in settings:
+            result += " %s" % domain
```

Comment:
Python's `join` is more succinct than looped concatenation. Could we do that instead?

Something like this maybe?
```
def convert_resolv_conf_search(settings):
    """Returns a settings string formatted for resolv.conf."""
    return "domain " + " ".join(settings) + "\n"
```

Reply:
I am nowhere close to being an expert, so I'll let you make that call.